src/task_7_1.py

```diff
@@ -0,0 +1,92 @@
+# Написать 12 функций по переводу:
+# 1. Дюймы в сантиметры
+# 2. Сантиметры в дюймы
+# 3. Мили в километры
+# 4. Километры в мили
+# 5. Фунты в килограммы
+# 6. Килограммы в фунты
+# 7. Унции в граммы
+# 8. Граммы в унции
+# 9. Галлон в литры
+# 10. Литры в галлоны
+# 11. Пинты в литры
+# 12. Литры в пинты
+# Примечание: функция принимает на вход число и возвращает конвертированное число
+# 2. Написать программу, со следующим интерфейсом: пользователю предоставляется на
+# выбор 12 вариантов перевода(описанных в первой задаче). Пользователь вводит цифру
+# от одного до двенадцати. После программа запрашивает ввести численное значение.
+# Затем программа выдает конвертированный результат. Использовать функции из первого
+# задания. Программа должна быть в бесконечном цикле. Код выхода из программы - “0”
+
+def perevodchik1 (n, m):
+    inch = 2.54
+    if m == 'duim':
+        print('{} inch = {} cm'. format(n,(n*inch)))
+perevodchik1 (int(input()), 'duim')
```

Comment:
Не нужно передавать внутрь функции отдельно строку 'duim' потому что она внутри никак не используется

```suggestion
perevodchik1 (int(input()))
```